### DISCUSSION TITLE
Can we use wdi5 for an application with luigi orchestration framework?

### DISCUSSION THREAD

**Discussion by mokashikushal** [ui5-community/wdi5#446] [Q&A] [answered]
Link to the luigi orchestration framework is 
https://luigi-project.io/

**Comment by vobu** [comment, marked as answer]
a quick glance through the Luigi docs showed that it has integration w/ UI5 → then `wdi5` fits right in to locate and interact with UI5 controls and UI5 Web Components.
that being said, `wdi5` is an extension to WebdriverIO `(wdio)` and `wdio` is capable of testing any kind of webapp, with presets for Vue, React, Angular and Svelte.
So I'd say Luigi and `wdi5`/`wdio` are a match 😄
